src/components/primary-button/PrimaryButton.stories.tsx

```diff
@@ -20,7 +20,7 @@ export const loading = () => <PrimaryButton isLoading>Click Here</PrimaryButton>
 export const disabled = () => <PrimaryButton isDisabled>Click Here</PrimaryButton>;
 
 export default {
-    title: 'Components|PrimaryButton',
+    title: 'Components|Buttons/PrimaryButton',
```

Comment:
Is the purpose of the `Buttons/` here for visual organization? These components are all top-level and aren't really grouped together at the moment.

Reply:
![image](https://user-images.githubusercontent.com/1571667/140570527-6e31167f-11d2-40a8-9027-b1e5253ce6d9.png)

It creates this grouping in the docs so you don't have to jump around to get to the PrimaryButton and PlainButton  docs, which inherit the API from the base Button